docs/Ansible-Connection.md

```diff
@@ -1,22 +1,22 @@
 # ansible Connection
 Ansible supports various connection plugins, you could get the list of the
-supported plugins by running 
+supported plugins by running
 ```
 ansible-doc --type connection -l
 ```
-The type of the Architecture you are adopting determines which connection
-you'll be using. For example if you setting up dhis2 on an single server using
-lxd containers, your connection will be 'lxd'. Secure Shell (ssh) is the
-default connection plugin for ansible, 
+The type of Architecture you are adopting determines which connection
+you'll be using. For example, if you're setting up dhis2 on an single server using
+lxd containers, your connection will be 'lxd'. However, Secure Shell (SSH) is the
+default connection plugin for ansible.
 
 ## lxd connection 
 When the setup is all in a single server, we separate components with lxd
 containers. To automate running tasks inside these containers with ansible,
-there is already a connection plugin available. Its lxd plugin. 
+there is already a connection plugin available. It's the [lxd plugin](https://docs.ansible.com/ansible/latest/collections/community/general/lxd_container_module.html). 
```

Comment:
I think you meant this link. https://docs.ansible.com/ansible/latest/collections/community/general/lxd_connection.html,